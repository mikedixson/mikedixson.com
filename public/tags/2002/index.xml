<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>2002 on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/2002/</link>
    <description>Recent content in 2002 on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Wed, 06 Aug 2014 14:48:46 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/2002/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Fixed: phpMyAdmin is throwing a #2002 cannot log in to the mysql server phpmyadmin</title>
      <link>https://mikedixson.com/fixed-phpmyadmin-is-throwing-a-2002-cannot-log-in-to-the-mysql-server-phpmyadmin/</link>
      <pubDate>Wed, 06 Aug 2014 14:48:46 +0000</pubDate>
      <guid>https://mikedixson.com/fixed-phpmyadmin-is-throwing-a-2002-cannot-log-in-to-the-mysql-server-phpmyadmin/</guid>
      <description>&lt;p&gt;This one had me going round the houses a bit until I the wrong permissions confirmed something to me.&lt;br&gt;&#xA;Firstly the &lt;code&gt;/etc/phpmyadmin/config.inc.php&lt;/code&gt; file is crud. For some reason this file doesn’t resemble the actual config file that you should use.&lt;br&gt;&#xA;The file &lt;code&gt;/usr/share/phpmyadmin/config.sample.inc.php&lt;/code&gt; is the format that you need for a correct phpMyAdmin configuration, so copy it to &lt;code&gt;/etc/phpmyadmin/config.inc.php&lt;/code&gt;&lt;/p&gt;&#xA;&lt;p&gt;Next you need to edit this config file. This isn’t the hardest of the things in the world to do if you just want the basics. You just need to edit the value for &lt;code&gt;$cfg[&#39;Servers&#39;][$i][&#39;host&#39;]&lt;/code&gt;&lt;/p&gt;&#xA;&lt;p&gt;Once you’ve done that you can then login to phpMyAdmin using your mySQL credentials and get started.&lt;br&gt;&#xA;It will bitch about not having the advanced features enabled. To do this follow the blog post I did at &lt;a href=&#34;http://mikedixson.com/2014/08/installing-phpmyadmin-advanced-features-on-ubuntu/&#34; title=&#34;INSTALLING PHPMYADMIN ADVANCED FEATURES ON UBUNTU&#34;&gt;INSTALLING PHPMYADMIN ADVANCED FEATURES ON UBUNTU&lt;/a&gt;&lt;/p&gt;&#xA;&lt;p&gt;Make sure you log out and back in to pick up the changes.&lt;/p&gt;&#xA;&lt;p&gt;Enjoy,&lt;br&gt;&#xA;Mike&lt;/p&gt;&#xA;</description>
    </item>
  </channel>
</rss>