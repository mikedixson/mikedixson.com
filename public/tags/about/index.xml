<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>About on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/about/</link>
    <description>Recent content in About on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Wed, 13 Jan 2010 12:00:22 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/about/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Publishing my thoughts before the mecons get them!</title>
      <link>https://mikedixson.com/publishing-my-thoughts-before-the-mecons-get-them/</link>
      <pubDate>Wed, 13 Jan 2010 12:00:22 +0000</pubDate>
      <guid>https://mikedixson.com/publishing-my-thoughts-before-the-mecons-get-them/</guid>
      <description>&lt;p&gt;As someone who is naturally very creative and also naturally very lazy I’ve always been one to have bright ideas but not to put them into action.&lt;/p&gt;&#xA;&lt;p&gt;It’s come to my attention lately that my ideas are becoming predictions and several of my ideas have crept into the real world without my participation. Now I’m by no means saying that pixies are coming and stealing my thoughts in the night but I wanted to make my ideas sort of open source. An open source idea would be one that anyone could act on but that I would be credited for coming up with the initial idea, no financial motives here, just pure ego.&lt;/p&gt;&#xA;&lt;p&gt;The following posts will detail an idea of mine as thoroughly as I can.&lt;/p&gt;&#xA;&lt;p&gt;MikeD&lt;/p&gt;&#xA;</description>
    </item>
  </channel>
</rss>