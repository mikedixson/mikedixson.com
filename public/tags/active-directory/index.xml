<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Active Directory on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/active-directory/</link>
    <description>Recent content in Active Directory on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Fri, 21 Sep 2018 12:27:08 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/active-directory/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>SOLVED: Windows does not have enough information to verify the certificate</title>
      <link>https://mikedixson.com/solved-windows-does-not-have-enough-information-to-verify-the-certificate/</link>
      <pubDate>Fri, 21 Sep 2018 12:27:08 +0000</pubDate>
      <guid>https://mikedixson.com/solved-windows-does-not-have-enough-information-to-verify-the-certificate/</guid>
      <description>&lt;h1 id=&#34;problem&#34;&gt;Problem&lt;/h1&gt;&#xA;&lt;p&gt;When moving from one ADFS server to another I imported a full certificate chain and private key into Machine certs on the new Windows 2016.&lt;br&gt;&#xA;The certificate displayed fine and chain is complete.&lt;/p&gt;&#xA;&lt;p&gt;&lt;strong&gt;However&lt;/strong&gt; viewing the certificate properties via ADFS the certificate information gives the error “Windows does not have enough information to verify the certificate”&lt;/p&gt;&#xA;&lt;h1 id=&#34;solution&#34;&gt;Solution&lt;/h1&gt;&#xA;&lt;p&gt;Move the intermediate certificate to the Intermediate Certificate Authorities &amp;gt; Certificates store&lt;/p&gt;&#xA;&lt;p&gt;It seems stupidly obvious in retrospect but Windows itself could view the cert fine, and the import was done via Machine Certificates so you’d think it would put things in the right place.&lt;/p&gt;&#xA;</description>
    </item>
    <item>
      <title>Get Boot Time For all computers in an OU – Powershell</title>
      <link>https://mikedixson.com/get-boot-time-for-all-computers-in-an-ou-powershell/</link>
      <pubDate>Tue, 13 Feb 2018 10:26:37 +0000</pubDate>
      <guid>https://mikedixson.com/get-boot-time-for-all-computers-in-an-ou-powershell/</guid>
      <description>&lt;p&gt;I wrote the following script enumerates an OU and then looks up the last boot time for each computer.&lt;/p&gt;&#xA;&lt;p&gt;&lt;a href=&#34;https://github.com/mikedixson/GetLastBootTimeFromOU&#34;&gt;https://github.com/mikedixson/GetLastBootTimeFromOU&lt;/a&gt;&lt;/p&gt;&#xA;&lt;p&gt;Note: I’m starting to put any scripts I write on GitHub, this will make it easier for you copy down and easier for me to keep any scripts up to date.&lt;/p&gt;&#xA;</description>
    </item>
    <item>
      <title>Exporting Active Directory Users Into CSV</title>
      <link>https://mikedixson.com/exporting-active-directory-users-into-csv/</link>
      <pubDate>Fri, 18 Dec 2015 19:03:06 +0000</pubDate>
      <guid>https://mikedixson.com/exporting-active-directory-users-into-csv/</guid>
      <description>&lt;p&gt;It took far too long to find this very very useful command online so I thought I’d repost to help others find it.&lt;br&gt;&#xA;The below command exports as follows:&lt;br&gt;&#xA;Connection to server: domain.local&lt;br&gt;&#xA;Exporting attributes: whenChanged,whenCreated,cn,sAMAccountName,givenName,sn&lt;br&gt;&#xA;From DN Root: DC=domain,DC=local&lt;br&gt;&#xA;Into File: domain.local-users.csv&lt;br&gt;&#xA;&lt;code&gt;csvde -s domain.local -l &amp;quot;whenChanged,whenCreated,cn,sAMAccountName,givenName,sn&amp;quot; -d &amp;quot;DC=domain,DC=local&amp;quot; -f domain.local-users.csv&lt;/code&gt;&lt;/p&gt;&#xA;</description>
    </item>
  </channel>
</rss>