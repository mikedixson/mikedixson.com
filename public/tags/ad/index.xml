<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Ad on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/ad/</link>
    <description>Recent content in Ad on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Tue, 13 Feb 2018 10:26:37 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/ad/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Get Boot Time For all computers in an OU – Powershell</title>
      <link>https://mikedixson.com/get-boot-time-for-all-computers-in-an-ou-powershell/</link>
      <pubDate>Tue, 13 Feb 2018 10:26:37 +0000</pubDate>
      <guid>https://mikedixson.com/get-boot-time-for-all-computers-in-an-ou-powershell/</guid>
      <description>&lt;p&gt;I wrote the following script enumerates an OU and then looks up the last boot time for each computer.&lt;/p&gt;&#xA;&lt;p&gt;&lt;a href=&#34;https://github.com/mikedixson/GetLastBootTimeFromOU&#34;&gt;https://github.com/mikedixson/GetLastBootTimeFromOU&lt;/a&gt;&lt;/p&gt;&#xA;&lt;p&gt;Note: I’m starting to put any scripts I write on GitHub, this will make it easier for you copy down and easier for me to keep any scripts up to date.&lt;/p&gt;&#xA;</description>
    </item>
    <item>
      <title>Exporting Active Directory Users Into CSV</title>
      <link>https://mikedixson.com/exporting-active-directory-users-into-csv/</link>
      <pubDate>Fri, 18 Dec 2015 19:03:06 +0000</pubDate>
      <guid>https://mikedixson.com/exporting-active-directory-users-into-csv/</guid>
      <description>&lt;p&gt;It took far too long to find this very very useful command online so I thought I’d repost to help others find it.&lt;br&gt;&#xA;The below command exports as follows:&lt;br&gt;&#xA;Connection to server: domain.local&lt;br&gt;&#xA;Exporting attributes: whenChanged,whenCreated,cn,sAMAccountName,givenName,sn&lt;br&gt;&#xA;From DN Root: DC=domain,DC=local&lt;br&gt;&#xA;Into File: domain.local-users.csv&lt;br&gt;&#xA;&lt;code&gt;csvde -s domain.local -l &amp;quot;whenChanged,whenCreated,cn,sAMAccountName,givenName,sn&amp;quot; -d &amp;quot;DC=domain,DC=local&amp;quot; -f domain.local-users.csv&lt;/code&gt;&lt;/p&gt;&#xA;</description>
    </item>
  </channel>
</rss>