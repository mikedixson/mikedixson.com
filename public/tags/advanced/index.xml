<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>ADVANCED on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/advanced/</link>
    <description>Recent content in ADVANCED on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Wed, 06 Aug 2014 14:35:25 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/advanced/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>INSTALLING PHPMYADMIN ADVANCED FEATURES ON UBUNTU</title>
      <link>https://mikedixson.com/installing-phpmyadmin-advanced-features-on-ubuntu/</link>
      <pubDate>Wed, 06 Aug 2014 14:35:25 +0000</pubDate>
      <guid>https://mikedixson.com/installing-phpmyadmin-advanced-features-on-ubuntu/</guid>
      <description>&lt;p&gt;Very concise and useful blog posting on the subject&lt;br&gt;&#xA;&lt;a title=&#34;INSTALLING PHPMYADMIN ADVANCED FEATURES ON UBUNTU&#34; href=&#34;http://www.pyrosoft.co.uk/blog/2008/04/30/installing-phpmyadmin-advanced-features-on-ubuntu/&#34; target=&#34;_blank&#34; rel=&#34;noopener&#34;&gt;INSTALLING PHPMYADMIN ADVANCED FEATURES ON UBUNTU&lt;/a&gt;&lt;/p&gt;&#xA;</description>
    </item>
  </channel>
</rss>