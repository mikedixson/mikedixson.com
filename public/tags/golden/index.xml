<?xml version="1.0" encoding="utf-8" standalone="yes"?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Golden on MikeDixson.com</title>
    <link>https://mikedixson.com/tags/golden/</link>
    <description>Recent content in Golden on MikeDixson.com</description>
    <generator>Hugo</generator>
    <language>en-gb</language>
    <lastBuildDate>Mon, 26 Nov 2012 08:06:26 +0000</lastBuildDate>
    <atom:link href="https://mikedixson.com/tags/golden/index.xml" rel="self" type="application/rss+xml" />
    <item>
      <title>Golden Hour â€“ Blackheath Church</title>
      <link>https://mikedixson.com/golden-hour/</link>
      <pubDate>Mon, 26 Nov 2012 08:06:26 +0000</pubDate>
      <guid>https://mikedixson.com/golden-hour/</guid>
      <description>&lt;figure style=&#34;width: 768px&#34; class=&#34;wp-caption alignnone&#34;&gt;[&lt;img loading=&#34;lazy&#34; class=&#34;size-full&#34; title=&#34;Blackheath Church 2013&#34; alt=&#34;Blackheath Church.jpg&#34; src=&#34;https://mikedixson.com/wp-content/uploads/2012/11/20121126-080625.jpg&#34; width=&#34;768&#34; height=&#34;1024&#34; /&gt;][1]&lt;figcaption class=&#34;wp-caption-text&#34;&gt;Blackheath Church&lt;/figcaption&gt;&lt;/figure&gt;&#xA;</description>
    </item>
  </channel>
</rss>